src/test/java/de/retest/web/selenium/ByBestMatchToRetestIdTest.java

```diff
@@ -0,0 +1,79 @@
+package de.retest.web.selenium;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import de.retest.ui.descriptors.Attributes;
+import de.retest.ui.descriptors.Element;
+import de.retest.ui.descriptors.IdentifyingAttributes;
+import de.retest.ui.descriptors.RootElement;
+
+class ByBestMatchToRetestIdTest {
+
+	@Test
+	void constructor_should_throw_new_illegal_argument_exception() {
```

Comment:
Test names should follow the [should pattern](https://stackoverflow.com/questions/155436/unit-test-naming-best-practices/7398606#7398606). That is, the test name should suggest what happens und which circumstances, e.g., `ctor_should_throw_new_illegal_argument_exception_when_retest_id_is_empty`.